src/_includes/layouts/default.njk

```diff
@@ -13,13 +13,7 @@
         <meta name="robots" content="index,follow">
         <meta name="generator" content="Eleventy">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-{#
-        {% if site.icon %}
-            <link rel="shortcut icon" type="image/png" href="{{ site.icon }}"/>
-        {% endif %}
-#}
-        {# TODO: figure out how to reference / generate local image #}
-        <link rel="shortcut icon" type="image/png" sizes="32x32" href="https://auth.fission.codes/favicon-32x32.png">
+        <link rel="icon" type="image/svg+xml" href="{{ "/images/favicon.svg"}}" sizes="any">
```

Comment:
does this path always resolve correctly on the IPFS version of the site? may need to use the `relativePath` filter: 
```suggestion
        <link rel="icon" type="image/svg+xml" href="{{ "/images/favicon.svg" | relativePath(page) }}" sizes="any">
```

Reply:
Ah yeah, it probably wouldn't. Updated it with the relative path filter.

Thanks for catching that. 🙏